test/torchaudio_unittest/functional/batch_consistency_test.py

```diff
@@ -241,3 +241,14 @@ def test_filtfilt(self):
         itemwise_output = torch.stack([F.filtfilt(x[i], a[i], b[i]) for i in range(self.batch_size)])
 
         self.assertEqual(batchwise_output, itemwise_output)
+
+    def test_apply_beamforming(self):
+        torch.random.manual_seed(2434)
+        sr = 8000
+        x = common_utils.get_whitenoise(sample_rate=sr, duration=0.05, n_channels=6)
+        specgram = common_utils.get_spectrogram(x, n_fft=400, hop_length=100)
+        specgram = specgram.view(2, 3, specgram.size(-2), specgram.size(-1))
+        beamform_weights = torch.rand(2, 201, 3, dtype=torch.cfloat)
```

Comment:
`201` is constrained on `n_fft=400` above, so can you define these parameters alongside `sr`?

Similarly, in my understanding 2 and 3 from L250 and L251 must be same, so can you define it as a variable?

This will make the code more intuitive to understand.

```python
sr = 8000
n_fft = 400
batch_size, num_channels = 2, 3
n_fft_bin = n_fft // 2 + 1
```
```python
x = common_utils.get_whitenoise(sample_rate=sr, duration=0.05, n_channels=batch_size * num_channels)
```